sway/handlers.c

```diff
@@ -344,13 +349,13 @@ static void handle_view_geometry_request(wlc_handle handle, const struct wlc_geo
 		view->desired_width = geometry->size.w;
 		view->desired_height = geometry->size.h;
 
-		if (view->is_floating) {
-			view->width = view->desired_width;
-			view->height = view->desired_height;
-			view->x = geometry->origin.x;
-			view->y = geometry->origin.y;
-			arrange_windows(view->parent, -1, -1);
-		}
+		/* if (view->is_floating) { */
+		/* 	view->width = view->desired_width; */
+		/* 	view->height = view->desired_height; */
+		/* 	view->x = geometry->origin.x; */
+		/* 	view->y = geometry->origin.y; */
+		/* /1* 	arrange_windows(view->parent, -1, -1); *1/ */
+		/* } */
```

Comment:
It's commented out for that reason, I didn't know what it might break to remove it. The issue is that arrange_windows calls wlc_view_set_geometry at some point and a call to `wlc_view_set_geometry` triggers the `handle_view_geometry_request`, so you end up with the bug described in: #487 and this bug appeared with all floating windows with the border code.

Now I know what it might break, so I can investigate. thanks :)